### PR TITLE
Fixed the description of ctrl + u

### PR DESCRIPTION
**Pull request by 374632897**
the shortcut cmd + u or  ctrl + u is used to delete the content of current cursor back to the start of line, not always delete the whole line

### CODE REVIEW

**Comment by jlevy** [comment]
LGTM. Thanks!
This PR also changes two other translations at once, though, so will need additional review.
@kevingo @Psycho7 can you review?

File: README-zh-Hant.md [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
 - 在 Bash 中，可以使用 **Tab** 自動補全參數，使用 **ctrl-r** 搜尋命令列歷史（在按下之後，鍵入便可以搜尋，重複按下**ctrl-r**會在更多匹配中迴圈，按下 **Enter** 會執行找到的命令，按下右方向鍵會將結果放入當前行中，使你可以進行編輯）。
 
-- 在 Bash 中，可以使用 **ctrl-w** 刪除你鍵入的最後一個單詞，使用 **ctrl-u** 刪除整行，使用 **alt-b** 和 **alt-f**以單詞為單位移動游標，​​使用 **ctrl-a** 將游標移至行首，使用 **ctrl-e** 將游標移至行尾，使用 **ctrl-k** 刪除游標至行尾的所有內容，使用 **ctrl-l** 清屏。鍵入`man readline` 檢視Bash 中的預設快捷鍵，內容很多。例如 **alt-.** 迴圈地移向前一個參數，以及 **alt-*** 展開通配符。
+- 在 Bash 中，可以使用 **ctrl-w** 刪除你鍵入的最後一個單詞，使用 **ctrl-u** 刪除當前光標所在位置之前的內容，使用 **alt-b** 和 **alt-f**以單詞為單位移動游標，​​使用 **ctrl-a** 將游標移至行首，使用 **ctrl-e** 將游標移至行尾，使用 **ctrl-k** 刪除游標至行尾的所有內容，使用 **ctrl-l** 清屏。鍵入`man readline` 檢視Bash 中的預設快捷鍵，內容很多。例如 **alt-.** 迴圈地移向前一個參數，以及 **alt-*** 展開通配符。
```
Comment (kevingo) [inline README-zh-Hant.md]: s/光標/游標

Comment (kevingo) [inline README-zh-Hant.md]: 鍵入`man readline` 檢視 Bash 中的預設快捷鍵

Bash 前空一格空白

**Comment by Psycho7** [comment]
@Can be merged after minor change made. As is mentioned above by kevingo.

**Comment by 374632897** [comment]
@kevingo @jlevy @Psycho7 Thanks for your review, well, some changes I've made .

**Comment by jlevy** [comment]
👍 Thanks @374632897 @kevingo @Psycho7 !

**Pull request merged**